src/WPF/ArcGISRuntime.WPF.Viewer/Resources/ControlStyles.xaml

```diff
@@ -1,12 +1,12 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
I am not a fan of the scrollbar styling, in context. The thumb blends into the background, and the trough stands out. I tried dragging the through at first...

![2018-03-23_11-00-18_arcgisruntime](https://user-images.githubusercontent.com/587809/37846088-c0563742-2e89-11e8-8e35-d911314ee460.png)